test/e2e/density.go

```diff
@@ -154,6 +154,14 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			// Verify latency metrics
+			// TODO: Update threshold to 1s once we reach this goal
+			// TODO: We should reset metrics before the test. Currently previous tests influence high percentiles
```

Comment:
The influence all percentiles, not only the "high" ones (although I agree that a difference can be the biggest for higher percentiles).
Can you please change it to just: "Currently previous tests influence latency metrics."?